### DISCUSSION TITLE
October 2024 Survey Results - Question 12

### DISCUSSION THREAD

**Discussion by boc-the-git** [mealie-recipes/mealie#4650] [General]
Placeholder discussion for people to share their thoughts and insights about the survey results.

**Comment by ChipSkylark37** [comment]
I wasn't able to participate in the survey, but comment some notes here:

1. Automatically Enter Edit Mode

95% of the time, I want to be able to edit it as I create it

2. Additional Options During Import

When I import a recipe, without scrolling down or adjusting things, I'd like to be able to have it prompt for certain things like 
- Do you want to import an image?
- Do you want to set a tag?
- Do you want to set required tools?

Or perhaps have that as an Advanced Import that has those as options, but not required.

3. Multiple Ingredient Units

I prefer to cook with grams / ml. A lot of recipes come as 'cups' or 'tsp' but I've found that, especially for baking, this can be too inaccurate. Currently, I just put the Grams measurement as a note field when using the ingredient amount. I do like the simplicity of the 'cups' let's say, but a short addition also include conversions would be helpful. The issue with using 'notes' is that it doesn't scale if I adjust the QTY up or down via scaling the recipe.

4. Option to have "Ingredient Units' on as Default

Since this is a requirement for scaling, I think it should be a user-defined option to have this enabled by default. I know that for a lot of recipes it may not import properly, but I'd rather set it up and fix it at time of import rather than converting it afterwards. 

5. Scaling Nutrition Values

I know there have been some other discussions and posts on this, but I'd love to see Nutrition Values scale as well.

6. "Want To Make" Option

I add a lot of different recipes and some I want to make more than others. I currently use a 'Tag' of 'Want To Make' so that I can filter based on this, but a button or something on a recipe to say "Hey, if I'm out of ideas and want to try something new, why not this one I marked". 

7. Sections Without Ingredients

You have to Toggle the "Section" for a specific ingredient. If you delete or move that ingredient, the Section goes along with it. For baking, I have a 'wet' 'dry' 'other' section for ingredients, so if I re-order something from Wet > Dry, it messes up the section and I have to toggle it off on one, on for another (if they are the top most one that the section is applying to). 

8. Showing Ingredients Per Step _without_ Cook Mode

Is there a reason we could not show the ingredients of a recipe, if linked to a step, when not in cook mode? I tend to use a laptop / surface device when in the kitchen and don't enter into Cook Mode, but would like to see which specific ingredients are being used. I understand I could just go into Cook Mode, but would like the option to stay in the default view.

9. Time Per Step / Time Remaining

If you associated a time to every step, you could then calculate 'time remaining' for the recipe. This would be extremely helpful for things like pies where you have to let it cool for a few hours. "Okay, I now have 3 hours to let cool, and if I check that off now, I can see that I should then have 5 hours remaining total" so you can plan the rest of your day. The 'Total Time / Prep Time' is okay, but '8 hours' is not great without understanding each of the steps and where you could take breaks.

==================================================

That's what comes to mind after using Mealie for some time, and would be great user workflow enhancements for me. A lot of this could be user-defined if you want it enabled or not. Perhaps in an 'advanced features' section where you can just toggle off / on the flags.
